pydy/utils.py

```diff
@@ -33,6 +33,12 @@ def sympy_equal_to_or_newer_than(version, installed_version=None):
     return parse_version(v) >= parse_version(version)
 
 
+def sympy_newer_than(version):
+    """Returns true if the installed version of SymPy is newer than the
+    provided version string."""
+    return parse_version(SYMPY_VERSION) > parse_version(version)
```

Comment:
Since setuptools if far from version 8 and SymPy has had the correct versions for over a year (maybe two). I think all this could be dropped.